pkgs/tools/misc/sheldon/default.nix

```diff
@@ -61,7 +61,6 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/rossmacarthur/sheldon";
     license = with licenses; [ mit ];
     maintainers = with maintainers; [ seqizz ];
-    platforms = platforms.linux;
```

Comment:
Platforms should still be set and to include darwin and other unix systems `platforms.unix` or to inclide non unix systems could also be set to `platforms.all`

```
platforms = platforms.unix;
```

Reply:
@jopejoe1 

Thank you for your review.
I think `platforms.unix` is better because sheldon has binary releases for linux and mac.
I will try building again with this fix.

Reply:
There's no reason to artificially restrict the platforms, IMO it should stay unset and use the defaults from the rust builder.

Reply:
Interesting, did not know that the rust builder sets them by default.